src/main/java/org/embl/mobie/io/ome/zarr/util/ZarrAxes.java

```diff
@@ -126,14 +93,23 @@ public boolean hasChannels() {
             this.axes.equals(TCZYX.axes);
     }
 
+    // the flag reverseAxes determines whether the index will be given w.r.t.
+    // reversedAxes=true corresponds to the java/bdv axis convention
+    // reversedAxes=false corresponds to the zarr axis convention
+    public int axisIndex(String axisName, boolean reverseAxes) {
```

Comment:
See my comment in mattermost chat. I am confused about this.

Reply:
I have answered in mattermost.

Reply:
I created an issue about this: https://github.com/mobie/mobie-io/issues/122.